lumicks/pylake/detail/plotting.py

```diff
@@ -0,0 +1,36 @@
+from matplotlib import pyplot as plt
+
+from ..adjustments import ColorAdjustment
+
+
+def get_axes(axes=None, image_handle=None):
+    """Return `axes` or the axes of the provided `image_handle` or ensure both axes are the same. If
+    neither `axes` nor `image_handle` are provided, fallback to the current `matplotlib` axes"""
+    if axes is None:
+        axes = plt.gca() if image_handle is None else image_handle.axes
+    elif image_handle and axes is not image_handle.axes:
+        raise ValueError("Supplied image_handle with a different axes than the provided axes")
```

Comment:
I think it would be nicer to `assert` rather than raise a `ValueError` here. The logic could be a tad simpler also:
```suggestion
    if axes is None:
        axes = plt.gca() if image_handle is None else image_handle.axes
    if image_handle:
            assert (
                axes == image_handle.axes
            ), "Supplied image_handle with a different axes than the provided axes"
```

Reply:
Yep, that is easier to grasp! Will change it.